packages/@contentful--app-scripts/bin/app-scripts

```diff
@@ -3,32 +3,41 @@
 const { program } = require('commander');
 const { version } = require('../package.json');
 
-const {createAppDefinition} = require('../')
+const {createAppDefinition, upload} = require('../')
 
-function getCommand(program, command) {
+
+async function runCommand(command, options) {
   const {ci} = program.opts()
-  return ci ? command.nonInteractive : command.interactive;
+  return ci ? await command.nonInteractive(options) : await command.interactive(options);
 }
```

Comment:
nit: I think this was good as it was: you're returning whatever the command returns, no need to `await`

Reply:
I always prefer adding await on every level. That way we get a proper stack trace. The outcome when running the code will be the same.

Reply:
I think in the end it doesn't really matter when in both cases `runCommand` will return a promise, or am I missing something ? @andipaetzold what was your intention to use await here?